src/backend/joanie/core/enums.py

```diff
@@ -0,0 +1,47 @@
+"""
+Core application enums declaration
+"""
+from django.utils.translation import gettext_lazy as _
+from django.utils.translation import pgettext_lazy
+
+PRODUCT_TYPE_CREDENTIAL = "credential"
+PRODUCT_TYPE_ENROLLMENT = "enrollment"
+PRODUCT_TYPE_CERTIFICATE = "certificate"
+
+PRODUCT_TYPE_CHOICES = (
+    # user purchases all (enrollments + certificate) from the start
+    (PRODUCT_TYPE_CREDENTIAL, _("Credential")),
+    # user is just enrolled in course runs (for free or not)
+    (PRODUCT_TYPE_ENROLLMENT, _("Enrollment")),
+    # user is enrolled in course runs for free
+    # finally orchestrator will offer user to pay to get a certificate
+    (PRODUCT_TYPE_CERTIFICATE, _("Certificate")),
+)
+
+ORDER_STATE_PENDING = "pending"  # waiting for payment and enrollment
+ORDER_STATE_CANCELED = "canceled"
+ORDER_STATE_FAILED = "failed"
+ORDER_STATE_IN_PROGRESS = "in_progress"
+ORDER_STATE_FINISHED = "finished"
+
+ORDER_STATE_CHOICES = (
+    (ORDER_STATE_PENDING, _("Pending")),  # default
+    (ORDER_STATE_CANCELED, pgettext_lazy("feminin", "Canceled")),
```

Comment:
I'm afraid this won't work because word's genres vary from language to language e.g. "le lait" :fr: but "la leche" :es:  :astonished: 

Maybe write the full context in english instead : 
```suggestion
    (ORDER_STATE_CANCELED, pgettext_lazy("As in: the order is cancelled.", "Canceled")),
```

https://twitter.com/loicsuberville/status/1359230310781751296

Reply:
"Bonjour Madame" 🤣